code/src/components/Header.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+
+import {
+  POPULAR_URL,
+  SCIFI_URL,
+  FRENCH_URL,
+  TOPRATED_URL,
+} from ".././urls";
+import "../css/header.css";
+
+export const Header = ({getUrl}) => {
+  const [prefix, setPrefix] = useState("POPULAR ");
+  
+  const checkPrefix = (url) => {
+    if (url === POPULAR_URL) setPrefix("POPULAR ");
+    else if (url === SCIFI_URL) setPrefix("SCI-FI ");
+    else if (url === FRENCH_URL) setPrefix("FRENCH ");
+    else setPrefix("TOP RATED ");
+  }
+  
+  const handleOnclick = (url) => {
+    getUrl(url);
+    checkPrefix(url);
+  }
+
+  return (
+    <header>
+      <h1>{prefix}MOVIES</h1>
+      <div className="buttons">
+        <button onClick = { () => handleOnclick(POPULAR_URL) } className="popular"><Link to="/">Popular</Link></button>
```

Comment:
When we want to pass function into an event handler like `onClick`, we should omit extra set of brackets after arrow function.

This is wrong piece of code :

```jsx
<button
  className="button"
  onClick={event => { onButtonClick(event) }}
>
  Click me!
</button>
```

This is correct piece of code :

```jsx
<button
  className="button"
  onClick={event => onButtonClick(event)}
>
  Click me!
</button>
```

Reply:
I’m not sure if I understand this comment, because I think I have done just like that?
i.e. I only have one set of curly brackets.

Reply:
You are right, I'm sorry! Whitespaces between equation mark mistaken me